src/main/java/com/feelfreecincinncati/service/iEventService.java

```diff
@@ -0,0 +1,10 @@
+package com.feelfreecincinncati.service;
+
+import com.feelfreecincinncati.dto.EventDTO;
+
+public interface iEventService {
```

Comment:
See comment above; make this IEventService.

Also, add JavaDoc to the interface.  It's more important to have it here than on the implementing classes.